mmpose/datasets/transforms/common_transforms.py

```diff
@@ -1027,6 +1027,17 @@ def transform(self, results: Dict) -> Optional[dict]:
                 if keypoint_weights:
                     encoded['keypoint_weights'] = keypoint_weights
 
+        if hasattr(self.encoder, 'field_mapping_table'
+                   ) and self.encoder.field_mapping_table is not None:
+            encoded['field_mapping_table'] = self.encoder.field_mapping_table
+        if hasattr(self.encoder, 'instance_mapping_table'
+                   ) and self.encoder.instance_mapping_table is not None:
+            encoded[
+                'instance_mapping_table'] = self.encoder.instance_mapping_table
+        if hasattr(self.encoder, 'label_mapping_table'
+                   ) and self.encoder.label_mapping_table is not None:
+            encoded['label_mapping_table'] = self.encoder.label_mapping_table
+
```

Comment:
How about setting default `field_mapping_table`, `instance_mapping_table` and `label_mapping_table` as empty dict in `BaseKeypointCodec`?